common/rabbitmq/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Rabbitmq CHANGELOG
 
 This file is used to list changes made in each version of the common chart rabbitmq.
 
+0.6.13
+------
+dusan.dordevic@sap.com
+-Adding standardized labels to all objects, according to https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels
```

Comment:
Please add the [Helm best practices for labels and annotations](https://helm.sh/docs/chart_best_practices/labels/) link too.

Reply:
Done